packages/compensation-change-review/tests/test_packet.py

```
@@ -0,0 +1,243 @@
+        "compensation_policy_reference": "compensation_policy",
+        "pay_equity_review_reference": "pay_equity_review",
+        "budget_authorization_reference": "budget_authorization",
+        "payroll_handoff_plan_reference": "payroll_handoff_plan",
+        "requester_reference": "actor",
+        "reviewer_reference": "actor",
+    }
+    digest_fields = [name for name in valid_kwargs() if name.endswith("_digest")]
+
+    for field_name, prefix in reference_fields.items():
+        with pytest.raises(ValueError, match=re.escape(f"opaque {prefix}: reference")):
+            replace(packet, **{field_name: f"wrong:11111111-1111-4111-8111-111111111111"})
```

Ruff (0.16.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)